tests/php/test_class_wc-connect-nux.php

```diff
@@ -7,56 +7,91 @@ public static function setupBeforeClass() {
+				)
+			), false
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => false,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => true,
+				)
+			), 'after_jetpack_connection'
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => true,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => true,
+				)
+			), 'after_jetpack_connection'
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => false,
+					'can_accept_tos'                  => true,
+					'should_display_after_cxn_banner' => false,
+				)
+			), 'tos_only_banner'
+		);
 	}
 
 	public function test_get_banner_type_to_display_no_jp_cxn_without_tos_acceptance() {
 		// before going through connection
 		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
 			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_NOT_INSTALLED,
 			'tos_accepted'                           => false,
-			'should_display_after_cxn_banner'        =>  false,
+			'can_accept_tos'                         => false,
+			'should_display_after_cxn_banner'        => false,
 		) ), 'before_jetpack_connection' );
 
 		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
 			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_INSTALLED_NOT_ACTIVATED,
 			'tos_accepted'                           => false,
-			'should_display_after_cxn_banner'        =>  false,
+			'can_accept_tos'                         => false,
+			'should_display_after_cxn_banner'        => false,
 		) ), 'before_jetpack_connection' );
 
 		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
 			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_ACTIVATED_NOT_CONNECTED,
 			'tos_accepted'                           => false,
-			'should_display_after_cxn_banner'        =>  false,
+			'can_accept_tos'                         => false,
+			'should_display_after_cxn_banner'        => false,
 		) ), 'before_jetpack_connection' );
 
 		// after going through connection, TOS was never accepted
 		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
 			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_CONNECTED,
 			'tos_accepted'                           => false,
-			'should_display_after_cxn_banner'        =>  true,
+			'can_accept_tos'                         => false,
+			'should_display_after_cxn_banner'        => true,
 		) ), 'after_jetpack_connection' );
 	}
 
```

Comment:
In addition to the `'can_accept_tos' => false` cases, we can add the true cases:

``` php
// before going through connection, user can accept tos
$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
  'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_NOT_INSTALLED,
  'tos_accepted'                           => false,
  'can_accept_tos'                         => true,
  'should_display_after_cxn_banner'        => false,
) ), 'before_jetpack_connection' ); // new test, passes as-is

$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
  'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_INSTALLED_NOT_ACTIVATED,
  'tos_accepted'                           => false,
  'can_accept_tos'                         => true,
  'should_display_after_cxn_banner'        => false,
) ), 'before_jetpack_connection' ); // new test, passes as-is

$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
  'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_ACTIVATED_NOT_CONNECTED,
  'tos_accepted'                           => false,
  'can_accept_tos'                         => true,
  'should_display_after_cxn_banner'        => false,
) ), 'before_jetpack_connection' ); // new test, passes as-is

// after going through connection, TOS was not already accepted before connection
$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
  'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_CONNECTED,
  'tos_accepted'                           => false,
  'can_accept_tos'                         => true,
  'should_display_after_cxn_banner'        => true,
) ), 'after_jetpack_connection' );  // new test, passes as-is

// after going through connection, TOS was already accepted before connection
$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
  'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_CONNECTED,
  'tos_accepted'                           => true,
  'can_accept_tos'                         => true,
  'should_display_after_cxn_banner'        => true,
) ), 'after_jetpack_connection' ); // new test, passes as-is

// make sure we get the right banner even in 'impossible' situations
$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
  'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_CONNECTED,
  'tos_accepted'                           => false,
  'can_accept_tos'                         => false,
  'should_display_after_cxn_banner'        => true,
) ), false ); // new test, does not pass
$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
  'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_CONNECTED,
  'tos_accepted'                           => true,
  'can_accept_tos'                         => false,
  'should_display_after_cxn_banner'        => true,
) ), false ); // new test, does not pass
```